Doc/library/hashlib.rst

```diff
@@ -301,6 +304,10 @@ a file or file-like object.
 
    .. versionadded:: 3.11
 
+   .. versionchanged:: next
+      Now raises a :exc:`BlockingIOError` if the file is opened in blocking
```

Comment:
Was this meant to be "`BlockingIOError` if the file is **not** opened in blocking mode" or "`BlockingIOError` if the file is opened in <strong>non-</strong>blocking mode."?

As written it contradicts the previous addition 30 lines up and the NEWS.d entry.

Reply:
Should be "if the file is opened in non-blocking mode."

More specifically:
If [`.readinto`](https://docs.python.org/3/library/io.html#io.RawIOBase.readinto) returns `None` because it has no data and would block raises `BlockingIOError`.

(PR to update the docs welcome)